staff.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Staff
+---
+
+## Identifying Staff
+
+The _staff_ are the volunteers who act as network and system administrators to
+keep Libera.Chat running smoothly.
+
+The complete current list of staff is available using the IRC command
+`/who libera/staff/*`.
+
+When this page was last updated, the staff members were:
+
+staff|staff|staff|staff
+--|--|--|--
+amdj     | emerson   | JonathanD  | nhandler
+bigpresh | emilsp    | jose       | nicole
+D[\_]     | Freso     | kline      | niko
+deedra   | Fuchs     | lausefuchs | njan
+edk      | glguy     | li         | Stx
+el       | jess      | mniip      | Swant
+
+Among the ops in `#libera` is also a service bot (which is not currently a
+member of staff):
+
+| bot
+|-
+| ozone
+
+## Contacting Staff
+
+### On IRC
+
+Assuming you can connect to IRC (`ircs://irc.libera.chat:6697`), that should be
+your first point of contact with staff.
+
+The quickest way to find the list of on-line staff is to look for chanops among
+the members in the `#libera` channel, and to find the complete list to
+
+Use the IRC command `/stats p` to get a list of available staff, and use
+`/msg`_`nick`_ to chat with one of them.
```

Comment:
```suggestion
`/msg <nick> <mesage>` to chat with one of them.
```
However it's relatively often advisable to use `/query` in clients that support it, if we're targeting inexperienced users that will generally open the privmsg window too, not just make it in the background

Reply:
Good catch, missing _message_.

However `/query` isn't available in all clients, so I guess I should document both.

I wasn't aware that in some clients `/msg` might _not_ open a window; it does in all the clients I've used.

Indeed it was many years after I started using IRC when I first heard about `/query`, which has always struck me as the most ridiculously mis-named command, but I'm often down-voted by people who've become accustomed to using it.

If we're targeting newbies, I think the use of `<>` as meta-characters is a bad idea, which I explain in #29.